application/src/test/java/org/togetherjava/tjbot/commands/tophelper/TopHelperMessageListenerTest.java

```diff
@@ -46,97 +43,84 @@ static void setUp() {
+        boolean shouldBeIgnored = topHelpersListener.shouldIgnoreMessage(event);
 
         // THEN the message is not ignored
         assertFalse(shouldBeIgnored);
     }
 
     @Test
-    @DisplayName("Ignores bots")
     void ignoresBots() {
         // GIVEN a message from a bot
         MessageReceivedEvent event =
-                createFakeMessageRecievedEvent(true, false, true, OVERVIEW_CHANNEL_PATTERN);
+                createFakeMessageReceivedEvent(true, false, true, OVERVIEW_CHANNEL_PATTERN);
 
         // WHEN checking if the message should be ignored
-        boolean shouldBeIgnored = topHelpersListenerSpy.shouldIgnoreMessage(event);
+        boolean shouldBeIgnored = topHelpersListener.shouldIgnoreMessage(event);
 
         // THEN the message is ignored
         assertTrue(shouldBeIgnored);
     }
 
     @Test
-    @DisplayName("Ignores webhooks")
     void ignoresWebhooks() {
         // GIVEN a message from a webhook
         MessageReceivedEvent event =
-                createFakeMessageRecievedEvent(false, true, true, OVERVIEW_CHANNEL_PATTERN);
+                createFakeMessageReceivedEvent(false, true, true, OVERVIEW_CHANNEL_PATTERN);
 
         // WHEN checking if the message should be ignored
-        boolean shouldBeIgnored = topHelpersListenerSpy.shouldIgnoreMessage(event);
+        boolean shouldBeIgnored = topHelpersListener.shouldIgnoreMessage(event);
 
         // THEN the message is ignored
         assertTrue(shouldBeIgnored);
     }
 
     @Test
-    @DisplayName("Ignores wrong channels")
     void ignoresWrongChannels() {
         // GIVEN a message outside a help thread
         MessageReceivedEvent eventNotAThread =
-                createFakeMessageRecievedEvent(false, false, false, OVERVIEW_CHANNEL_PATTERN);
+                createFakeMessageReceivedEvent(false, false, false, OVERVIEW_CHANNEL_PATTERN);
         MessageReceivedEvent eventWrongParentName =
-                createFakeMessageRecievedEvent(false, false, true, "memes");
+                createFakeMessageReceivedEvent(false, false, true, "memes");
 
         // WHEN checking if the message should be ignored
-        boolean ignoresNonThreadChannels =
-                topHelpersListenerSpy.shouldIgnoreMessage(eventNotAThread);
+        boolean ignoresNonThreadChannels = topHelpersListener.shouldIgnoreMessage(eventNotAThread);
         boolean ignoresWrongParentNames =
-                topHelpersListenerSpy.shouldIgnoreMessage(eventWrongParentName);
+                topHelpersListener.shouldIgnoreMessage(eventWrongParentName);
 
         // THEN the message is ignored
         assertTrue(ignoresNonThreadChannels, "Failed to ignore non-thread channels");
         assertTrue(ignoresWrongParentNames, "Failed to ignore wrong parent channel names");
     }
 
 
-    MessageReceivedEvent createFakeMessageRecievedEvent(boolean isBot, boolean isWebhook,
+    MessageReceivedEvent createFakeMessageReceivedEvent(boolean isBot, boolean isWebhook,
```

Comment:
i would drop the `Fake`, sounds a bit weird. we all know that this is a mocked event, its a unit test after all. and if at all, prefer a suffix like `...Mock` over `Fake`, more idiomatic. but id drop it all together actually.